test/e2e/framework/config.go

```diff
@@ -0,0 +1,60 @@
+// +build e2e
+
+package framework
+
+import (
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Config holds global test configuration translated from environment variables
+type Config struct {
+	SubscriptionID                    string `envconfig:"SUBSCRIPTION_ID"`
+	AzureClientID                     string `envconfig:"AZURE_CLIENT_ID"`
+	AzureClientSecret                 string `envconfig:"AZURE_CLIENT_SECRET"`
+	AzureTenantID                     string `envconfig:"AZURE_TENANT_ID"`
+	KeyvaultName                      string `envconfig:"KEYVAULT_NAME"`
+	Registry                          string `envconfig:"REGISTRY" default:"mcr.microsoft.com/oss/azure/secrets-store"`
+	ImageName                         string `envconfig:"IMAGE_NAME" default:"provider-azure"`
+	ImageVersion                      string `envconfig:"IMAGE_VERSION" default:"0.0.11"`
+	IsSoakTest                        bool   `envconfig:"IS_SOAK_TEST" default:"false"`
+	IsWindowsTest                     bool   `envconfig:"TEST_WINDOWS" default:"false"`
+	IsKindCluster                     bool   `envconfig:"CI_KIND_CLUSTER" default:"true"`
+	SecretValue                       string `envconfig:"SECRET_VALUE" default:"test"`
+	KeyValue                          string `envconfig:"KEY_VALUE" default:"uiPCav0xdIq"`
```

Comment:
I just want to make sure this wasn't an accident that you left the `KeyValue` default value still populated

Reply:
That's just to default to a value similar to how we do it today with the bats file.